advisor/src/main/kotlin/advisors/Osv.kt

```diff
@@ -58,7 +58,7 @@ class Osv(name: String, config: OsvConfiguration) : AdviceProvider(name) {
     class Factory : AbstractAdviceProviderFactory<Osv>("OSV") {
         override fun create(config: AdvisorConfiguration) =
             // OSV does not require any dedicated configuration to be present.
-            Osv(type, config.forProvider { osv })
+            Osv(type, config.forProvider { osv ?: OsvConfiguration() })
```

Comment:
Sorry for breaking this. Looks like this was broken also for `OssIndex` (from where I took the code) before. Mind finding it for `OssIndex`, too?

Reply:
Done.